deploy/roles/r-studio-server/defaults/main.yml

```diff
@@ -1,3 +1,8 @@
 ---
 rstudio_server_package_name: http://download2.rstudio.org/rstudio-server-rhel-1.1.383-x86_64.rpm
 rstudio_server_package_state: present
+
+rstudio_pro_drivers_url: http://drivers.rstudio.org/7C152C12/odbc-install.sh
+rstudio_pro_drivers_folder: /etc/rstudio
+rstudio_pro_driver_install_name: odbc-install.sh
+rstudio_pro_driver_install_folder: /usr/lib
```

Comment:
@stirlingv This is where I was thinking /opt/rstudio-server might make sense, perhaps /opt/rstudio-server/drivers?  You would also need a task to ensure that this directory exists and has the right permissions.